### ISSUE TITLE
Accessibility code causing input lag

### ISSUE THREAD

**Issue by paulcbetts** [atom/electron#4001]
It appears that #3336 or #3323 (not sure which yet) are causing serious input lag / typing performance regressions on Windows on certain machines. Timeline traces will show a significant amount of JS CPU time spent in "Other" while typing. You can also see these messages spammed constantly in console

```
[2476:0105/200323:WARNING:browser_accessibility_manager_win.cc(96)] Not firing AX event because of no hwnd
[2476:0105/200323:WARNING:browser_accessibility_manager_win.cc(178)] Not firing AX event because of no root_delegate or hwnd
[2476:0105/200323:WARNING:browser_accessibility_manager_win.cc(96)] Not firing AX event because of no hwnd
[2476:0105/200323:WARNING:browser_accessibility_manager_win.cc(178)] Not firing AX event because of no root_delegate or hwnd
[2476:0105/200323:WARNING:browser_accessibility_manager_win.cc(96)] Not firing AX event because of no hwnd
[2476:0105/200323:WARNING:browser_accessibility_manager_win.cc(178)] Not firing AX event because of no root_delegate or hwnd
[2476:0105/200323:WARNING:browser_accessibility_manager_win.cc(96)] Not firing AX event because of no hwnd
```

I'm going to look into this bug more and see if we can fix up the typing lag while still leaving a11y enabled

**Comment by paulcbetts** [comment]
#3323 looks okay, it's definitely #3336, gonna see if I can fix this.

**Comment by joaomoreno** [comment]
:+1:

**Comment by ryanmolden** [comment]
@joaomoreno , I recall trying without that code in and you said something didn't work (maybe something to do with the mouse), do you recall exactly?

**Comment by joaomoreno** [comment]
Don't remember. We'd have to give it another try to find out.

**Comment by paulcbetts** [comment]
From the perspective of the Slack app, as long as you don't disable the intermediate window you got content out, though it was garbage either way because Slack sucks at accessibility right now.

**Comment by zcbenz** [comment]
Fixed with #4009.
